src/ui/title/Title.module.css

```diff
@@ -0,0 +1,13 @@
+.titleWrap {
+    font-size: 3.6rem;
+    font-weight: 500;
+    color: var(--color-eerie-black);
+    margin-bottom: 25px;
```

Comment:
try to avoid odd values here and in other pales even if they are described in the figma layout